velox/type/tests/VariantTest.cpp

```diff
@@ -180,11 +200,31 @@ TEST(VariantTest, longDecimal) {
   // 1.2345 < 12.345
   EXPECT_LT(
       u1.value<TypeKind::LONG_DECIMAL>(), v.value<TypeKind::LONG_DECIMAL>());
+  EXPECT_LE(
```

Comment:
Can you add combinations of positive and negative values.?
+, -
-, +
-, -

Reply:
Added these combinations with different (precision, scale), thanks!